Dockerfile

```diff
@@ -1,12 +1,17 @@
-ARG ARCH_ORG
+# We must use a glibc based distro due to embedded python not supporting musl libc for aarch64 (only amd64+musl is supported)
+# see https://github.com/indygreg/python-build-standalone/issues/87
+# use `docker buildx imagetools inspect cgr.dev/chainguard/wolfi-base:latest` to find latest sha256 of multiarch image
+ARG WOLFI_DIGEST=sha256:5cc4736f5a03e9ba618760e5b21c1f1bc0ce275fc6ba86589c8dd26374c2647b
+FROM --platform=$TARGETPLATFORM cgr.dev/chainguard/wolfi-base@$WOLFI_DIGEST
```

Comment:
Why use a `WOLFI_DIGEST` arg? If you in-line it, I think Dependabot will suggest upgrades. Sadly, you have to pay for tags of the Wolfi images.....

Reply:
Will it also take into account multi arch?

Reply:
I don't know. But keeping a digest up-to-date manually will never work in the long run.

Reply:
Removed the intermediate ARG and also enable dependabot for docker images.